### ISSUE TITLE
Ability to customize how coverage is reported on cpancover.com

### ISSUE THREAD

**Issue by haarg** [pjcj/Devel--Cover#113]
Some dists aren't fully covered by their standard tests in t, but only by extra tests that may be in a separate directory, or conditional on an environment variable.

Moo is one example of this.  The [report on cpancover.com](http://cpancover.com/latest/Moo-1.006001/index.html) shows the overall statement coverage as 90%, with some modules going as low as 40%.  There are also some modules not listed on the report at all.  If Moo's xt tests are run as well, the statement coverage will be 100%.

I would like some way to customize how cpancover.com is doing the coverage reporting.  Having a documented environment variable set would probably be be the simplest thing.  Then the Makefile.PL/Build.PL could adjust the dependencies and tests to be run as needed.

**Comment by pjcj** [comment]
Yes, something along these lines is important.  Thanks for noting it.
